xmodule/x_module.py

```diff
@@ -1844,7 +1846,7 @@ def render(self, block, view_name, context=None):
 
         """
         context = context or {}
-        return self.__getattr__('render')(block, view_name, context)
+        return self.__getattr__('render')(block, view_name, context)  # pylint: disable=unnecessary-dunder-call
```

Comment:
Refactoring `self.__getattr__('render')` to `getattr(self, 'render')` generates `C7630: literal-used-as-attribute` warning and using `self.render` instead causes the tests to go into infinite recursion.